NumberValidator/Validators/IN/AADHAAR.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Linq;
+using NumberValidator.Helpers;
+using System.Text.RegularExpressions;
+
+namespace NumberValidator.Validators.AADHAAR
+{
+    public class AADHAAR : IValidator
+    {
+        public bool IsValid(string aadhaar)
+        {
+            try
+            {
+                Validate(aadhaar);
+                return true;
+            }
+            catch
+            {
+                return false;
+            }
+        }
+        public void Validate(string aadhaar)
+        {
+            String aadhaarPattern = @"^[2-9][0-9]{11}$";
+
+            if (aadhaar.Length != 12)
+            {
+                throw new InvalidLengthException();
+            }
+            if (aadhaar.SequenceEqual(aadhaar.Reverse()))
+            {
+                throw new InvalidFormatException();
+            }
+            Regex aadhaarRegex = new Regex(aadhaarPattern);
```

Comment:
extra line between a code block which is ending and another when a different code block starts.